README.md

```diff
@@ -28,7 +28,7 @@ A bundle will be produced from the native packager settings of this project. A b
 packager distribution and includes some component configuration. To load the bundle:
 
 ```bash
-loadBundle <HIT THE TAB KEY AND THEN RETURN>
+conductr:loadBundle <HIT THE TAB KEY AND THEN RETURN>
```

Comment:
You're right that it's doable with commands, was not the immediate "sbt style" I would think of. 
It's more work but can be done if you think it's worth it